src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -10,6 +10,12 @@ redirect_from:
 description: "Learn about Sentry dynamically prioritizes your transaction data."
 ---
 
+<Note>
+
+Please note that Dynamic Sampling is only available on the latest version of our plans and starting from 5M reserved transactions. Your plan can be updated by any Organization Owner or Billing Member by going to Settings > Subscription > Manage Subscription
+
+</Note>
```

Comment:
```suggestion
<Note>

Dynamic Sampling is only available on the latest version of our <a class href=“https://sentry.io/pricing/”>plans</a>, starting from 5M reserved transactions. Anyone in your org with owner or billing-level permissions can update your plan by going to Settings > Subscription, and clicking the "Manage Subscription" button.

</Note>
```

Reply:
I wouldn't include a link to the pricing/plans page, since there is no helpful information for them there I'm afraid. Since the new plans are called exactly the same as the old, there's no way to distinguish :/ the rest of the suggestion is fine for me!